templates/azure/ebook/index.html

```diff
@@ -10,100 +10,73 @@
+        <div class="p-image-container u-hide--small">
+          <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/d5af1f9f-ebook-azure.png"
+            alt="Ubuntu pro and Azure logo" />
         </div>
       </div>
-      <div class="col-5 u-hide--small u-vertically-center u-align--center">
-        {{ image(url="https://assets.ubuntu.com/v1/0959ce33-NEW_Ubuntu_ebook_1.png",
-                alt="",
-                width="214",
-                height="300",
-                hi_def=True,
-                loading="auto",
-                attrs={ "class": "p-image--bordered"}) | safe
-        }}
-      </div>
     </div>
-  </section>
+  </div>
+</section>
 
   {% include "azure/shared/_azure-logo-strip.html" %}
 
-  <div class="p-strip--light">
-    <div class="row">
-      <div class="col-7">
-        <h2 class="p-heading--4">
-          Ubuntu Pro for Azure is a premium image delivering the most comprehensive open source security and compliance
-        </h2>
-
-        <hr class="p-separator u-no-margin--top" />
-
-        <p class="p-heading--4">
-          <strong>70% of Azure virtual machines are running on Ubuntu due to reliability and ease of use</strong>
-        </p>
-
-        <p>
-          As the publishers of Ubuntu, we went one step further and developed a premium Ubuntu Pro image in partnership with public cloud providers, to deliver a preferred open-source solution for cloud innovators.
-        </p>
-
-        <p>
-          Here is our latest eBook, where we share the details of how Ubuntu Pro on Azure delivers extended open-source security, enterprise compliance, 10 years of extended maintenance and more.
-        </p>
-      </div>
-
-      <div class="col-5" id="ubuntu-pro-ebook">
-        {% with id="4393", returnURL="/azure/ebook/thank-you", cta="Get the eBook", cta_class="p-button--positive" %}
-          {% include "engage/shared/_en_engage_form.html" %}
-        {% endwith %}
-      </div>
-    </div>
+<section class="p-section">
+  <div class="u-fixed-width p-section--shallow">
+    <hr />
+    <h2>Ubuntu Pro for Azure is a premium image delivering the most comprehensive open source security and compliance</h2>
   </div>
-
-  <div class="p-strip">
-    <div class="row--50-50">{% include "azure/shared/_how-ubuntu-pro-helps-you.html" %}</div>
+  <div class="row--50-50">
+    <div class="col">
+      <p class="p-heading--5">70% of Azure virtual machines are running on Ubuntu due to reliability and ease of use</p>
+      <p>As the publishers of Ubuntu, we went one step further and developed a premium Ubuntu Pro image in partnership with public cloud providers, to deliver a preferred open-source solution for cloud innovators.</p>
+      <p>Here is our latest eBook, where we share the details of how Ubuntu Pro on Azure delivers extended open-source security, enterprise compliance, 10 years of extended maintenance and more.</p>
+    </div>
+    <div class="col" id="ubuntu-pro-ebook">
```

Comment:
So that it doesn't scroll to the middle of the form 
```suggestion
    <div class="col" id="ubuntu-pro-ebook" style="scroll-margin-top: 4rem">
```